RELEASE/scripts/autoscend.ash

```diff
@@ -2299,11 +2299,6 @@ boolean questOverride()
 		}
 	}
 
-	if(get_property("auto_hippyInstead").to_boolean())
-	{
-		set_property("auto_ignoreFlyer", true);
-	}
-
```

Comment:
I would just remove the `questOverride()` function entirely. The last code left in there looks like it was something Cheesecookie wrote probably when mafia was not as reliable as it is now as it seems to do nothing that mafia doesn't already.

Reply:
done at commit 10a89ca

Reply:
oops, forget to remove it from header too.
commit ddcc1e7